src/ConfigSetup.h

```diff
@@ -379,7 +390,9 @@ class ConfigSetup
   config_setup::SystemVals sys;
   ConfigSetup(void);
   void Init(const char *fileName);
-
+  #if GOMC_LIB_MPI
+  void Init(const char *fileName, MultiSim *& multisim);
+  #endif
```

Comment:
@ynejahi @go2432 can we use one Init function, like this?

`void Init(const char *fileName, MultiSim=NULL);`

and in cpp file, check if it was not NULL, we assign the vectors of temperature, pressure, and chempot

Reply:
Implemented this like so
  void Init(const char *fileName, MultiSim const*const& multisim);